__tests__/api.test.js

```diff
@@ -123,8 +123,42 @@ describe('/api/articles/:article_id/comments testing', () => {
             })
         })
     })
+    test('returned comment array should have the most recent comments first', () => {
+        return request(app)
+        .get('/api/articles/1/comments')
+        .expect(200)
+        .then((response) => {
+            const comments = response.body.comments;
+            expect(comments).toBeSortedBy('created_at', {descending: true})
+        })
+    })
+    test('returns 200 with the message "There are no comments about this article" when article_id is valid and the article exists but no comments are in the array', () => {
+        return request(app)
+        .get('/api/articles/2/comments')
+        .expect(200)
+        .then((response) => {
+            const msg = response.body.msg;
+            expect(msg).toBe('There are no comments about this article')
```

Comment:
It would be better to just return an empty array here, it will be easier to handle when you come to make a front end for this app :)